### PR TITLE
Battery Parametrized Bidder

### PR DESCRIPTION
**Pull request by dguittet**

### CODE REVIEW

**Comment by dguittet** [comment]
@bknueven could you please take a look at this battery parametrized bidder?

File: dispatches/case_studies/renewables_case/prescient_options.py [hunk 1]
```diff
@@ -15,9 +15,10 @@
...
 
 solvers_list = ["gurobi_direct", "xpress_direct", "cbc"]
-opt = None
+opt = False
 for solver_name in solvers_list:
     if pyo.SolverFactory(solver_name).available(exception_flag=False):
+        opt = True
         break
 if not opt:
     raise RuntimeWarning("No available solvers")
```
Comment (bknueven) [inline dispatches/case_studies/renewables_case/prescient_options.py]: Not really relevant to this PR, but this code could use Pythons `for/else` construct in place of the boolean `opt`.

**Review by bknueven — APPROVED** [review]
Looks good and makes sense

**Comment by dguittet** [comment]
@Xinhe-Chen I fixed the solver_name thing so it should default to 'gurobi' for you now without you having to set it in every file

**Pull request merged**